x/concentrated-liquidity/bench_test.go

```diff
@@ -36,10 +36,12 @@ func (s *BenchTestSuite) createPosition(accountIndex int, poolId uint64, coin0,
 }
 
 func noError(b *testing.B, err error) {
+	b.Helper()
```

Comment:
what does this do?

Reply:
marks the function as a benchmark helper function so that the tooling can give additional feedback. 

https://github.com/kulti/thelper

```
b_begin - b.Helper() should begin helper function.
```

https://cs.opensource.google/go/go/+/go1.21.0:src/testing/testing.go;l=1127